### ISSUE TITLE
Touch Bar integration broken (again)

### ISSUE THREAD

**Issue by jpcranford** [atom-community/tool-bar#325]
Sometime between the fixes contained in #312 (i.e. a set of curly brackets in the /lib/touch-bar-manager.js file) and the release of v1.4.1, the support for Touch Bars broke... I think. I uninstalled and reinstalled this plugin and the issue persists.

Honestly I'm out of my depth, and if there's a debug/reloading step to take, please let me know.

**Comment by aminya** [comment]
@jpcranford Could you run `apm install tool-bar` and post the output?

**Comment by jpcranford** [comment]
Okay, after it gave me some grief because of too many aliases, I fixed that and finally got this result:

```zsh
username_redacted@computer ~ % apm install tool-bar
Installing tool-bar to /Users/ldsmadman/.atom/packages ✓
```

**Comment by aminya** [comment]
This shows that the package is installed correctly. 

@suda Did you build-commit before publishing?

**Comment by suda** [comment]
@aminya yep, it's now a part of my release process.

I no longer have an MBP with a touch bar so I'm not able to debug it either. I'll mark this as "help needed" in case anyone else can help.

**Comment by ThatXliner** [comment]
I have the same issue too

**Comment by GRA0007** [comment]
Note that this is being addressed here: https://github.com/suda/tool-bar/pull/329

**Comment by aminya** [comment]
Fixed in 1.4.4
